plugins/gatsby-theme-iterative-docs/package.json

```diff
@@ -4,8 +4,31 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx ."
   },
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
I looked through the `gatsby-theme-iterative-docs` and spotted some dependences that aren't here:

* [github-slugger](https://github.com/iterative/dvc.org/blob/master/plugins/gatsby-theme-iterative-docs/createPages.js#L1)
* [lodash](https://github.com/iterative/dvc.org/blob/master/plugins/gatsby-theme-iterative-docs/src/components/Documentation/TutorialsLinks/index.tsx#L3)
* [title-case](https://github.com/iterative/dvc.org/blob/master/plugins/gatsby-theme-iterative-docs/src/utils/shared/sidebar.js#L23)
* [gatsby-link](https://github.com/iterative/dvc.org/blob/master/plugins/gatsby-theme-iterative-docs/src/components/Link/index.tsx#L4)
* [react-collapse](https://github.com/iterative/dvc.org/blob/master/plugins/gatsby-theme-iterative-docs/src/components/Documentation/Layout/SidebarMenu/Item/index.tsx#L2)

Do they need to be added to the dependencies?

Reply:
You're right! One thing to note is that `gatsby-link` should actually be removed as it's recommended to use the export from the main `gatsby` package, as per [the README for `gatsby-link`](https://www.npmjs.com/package/gatsby-link).

I'll add these as well as make the `gatsby-link` change.

Also, having both `react-collapsible` and `react-collapse` confused me a bit.